lib/httparty/request.rb

```diff
@@ -92,6 +92,11 @@ def uri
 
         path.path = "/#{path.path}" if last_uri_host[-1] != "/"
         path.path = last_uri_host + path.path
+
+        # use absolute path [#582]
```

Comment:
I think this comment is unnecessary, people could you `git blame` to figure out why this change was added

Reply:
Removed